examples.py/3D/Form/bricktower.py

```diff
@@ -15,35 +15,36 @@
 def setup():
     size(640, 360, OPENGL)
 
+
 def draw():
-  background(0)
-  (tempX, tempY, tempZ) = (0, 0, 0)
-  fill(182, 62, 29)
-  noStroke()
-  # Add basic light setup
-  lights()
-  translate(width / 2, height * 1.2, -380)
-  # Tip tower to see inside
-  rotateX(radians(-45))
-  # Slowly rotate tower
-  rotateY(frameCount * PI / 600)
-  for i in range(brickLayers):
+    background(0)
+    (tempX, tempY, tempZ) = (0, 0, 0)
```

Comment:
For some1 coming from another programming language and just learning about the differences about tuples & lists, `(tempX, tempY, tempZ) = (0, 0, 0)` seems more like a tuple assignment.
`tempX = tempY = tempZ = 0` has a much clearer intent for any programming language iMO.